### DISCUSSION TITLE
Is it possible to serialize the parser itself to a file?

### DISCUSSION THREAD

**Discussion by Atlinx** [b3b00/csly#361] [Q&A]
I was wondering if it was possible to serialize the parser itself to a file and then quickly load it so it doesn't have to rebuild the parser every time the application loads.

**Comment by b3b00** [comment]
Hello , sorry  there is no native way to serialize the parser.  I am not sure that the parser build time is such an issue that we need a way to load it faster. Mauve have you a particular use case ?

**Reply by Atlinx** [reply]
Ah no worries -- I was just curious and don't have a use case at the moment.
